jekyll/_cci2/deploy-android-applications.adoc

```diff
@@ -196,12 +220,12 @@ Now, you need to configure `.circleci/config.yml` to decrypt your keystore, gene
 If you have not already done so, create a `.circleci` folder in the root of your project repository. Inside the `.circleci` folder, create a `config.yml` file.
 
 [#set-up-config-with-the-android-orb]
-== 4a. Set up config with the Android orb
+== 4a. Set up configuration with the Android orb
```

Comment:
I think this bit looks slightly odd and could be easier to read using:

4. Configure deployment
    a. Using the Android orb
        i. step one
        ii. step two
    b. Without the Android orb
        i. step one
        .... etc etc.
 
But it definitely works as is so I am approving this PR regardless 🎉

Reply:
GH won't let be indent the a. b. i. ii. steps in a comment so that ^ looks odd but hopefully it makes sense.